pkg/testutil/events.go

```diff
@@ -402,6 +409,12 @@ func (ape GroupedEventsByID) Less(i, j int) bool {
 			return false
 		}
 		return ape[i].DeleteEvent.Identifier.String() < ape[j].DeleteEvent.Identifier.String()
+	case event.WaitType:
```

Comment:
I believe the WaitEvents are actually entirely predictable. So we might not need to sort them for test comparisons.